openquake/server/views.py

```diff
@@ -1516,6 +1516,27 @@ def download_aggrisk(request, calc_id):
     return response
 
 
+@cross_domain_ajax
+@require_http_methods(['GET'])
+def show_aggrisk_tags(request, calc_id):
+    job = logs.dbcmd('get_job', int(calc_id))
```

Comment:
Let's have a generic function called `show_dataframe` with arguments `(request, calc_id, name)` calling inside
```python
df = _extract(ds, name)
```

Reply:
Perhaps `extract_html_table` is a better name